### DISCUSSION TITLE
Customize disabled checkbox?

### DISCUSSION THREAD

**Discussion by florianmartens** [chakra-ui/chakra-ui#5791] [Q&A]
Simple question here. I'd like to customize the the background color of a disabled checkbox when its `checked`:

```
 <Checkbox
      disabled
      isDisabled={true}
      isFocusable={false}
      isReadOnly={true}
      size="lg"
  />
```

I can't pass the background color as a systems prop directly since the checkbox is a muli-part component. The parts are `icon`, `label` and `control` and I would like to change the background color of `control` whenever the checkbox is checked. I know this is possible when overwriting the theme but this would be overkill in this instance. Is there a way to do this?

**Comment by chr-ge** [comment]
You must modify the theme as it's the only way to access the `control` part:

```ts
const theme = extendTheme({
  components: {
    Checkbox: {
      parts: ["control"],
      baseStyle: {
        control: {
          _checked: {
            _disabled: {
              bg: "red.100"
            }
          }
        }
      }
    }
  }
});
```
